features/support/env.rb

```diff
@@ -55,6 +52,31 @@
   end
 end
 
+AfterAll do
```

Comment:
You might want to use `at_exit` (a Ruby rather than Cucumber thing) instead, as it should still get called in a Ctrl-C or Buildkite agent termination situation.

Reply:
Nice 👍